### DISCUSSION TITLE
Is it possible to show candidates through Ivy/Counsel?

### DISCUSSION THREAD

**Discussion by alienbogart** [company-mode/company-mode#1201] [Q&A] [answered]
### Runtime Environment

- OS: MX Linux 18.3
- [dotfiles](https://github.com/mrbig033/dotfiles)
- GNU Emacs 27.0.50
- Org mode version 9.2.3
- Evil version 1.2.14
- Company version: 0.9.10
- [~/.emacs.d](https://github.com/mrbig033/emacs.d)

### Issue

If this is not possible, I leave that as a suggestion as it would be quite useful. Thanks.

**Comment by clemera** [comment, marked as answer]
You can use `counsel-company` which comes with the `counsel` package.

**Reply by yugaego** [reply]
For reference: [counsel-company](https://github.com/abo-abo/swiper/blob/7cdde66c95d5205287e88010bc7a3a978c931db0/counsel.el#L355) is a part of the [Swiper](https://github.com/abo-abo/swiper) project.
